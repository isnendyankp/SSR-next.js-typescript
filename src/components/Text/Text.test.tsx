/*
    this is a test file for the Text component
*/

import Text from ".";
import { render, screen } from "@testing-library/react";

describe('Text component unit testing', () => {

// this test for text component with explanation should render text with children
    it('should render text with children', () => {
        render(<Text>Text with children</Text>);
        const textElement = screen.getByText(/text with children/i);
        expect(textElement).toMatchSnapshot();
    });
}
);
