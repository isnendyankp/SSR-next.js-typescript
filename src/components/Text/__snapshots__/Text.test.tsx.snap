// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component unit testing should render text with children 1`] = `
<p>
  Text with children
</p>
`;
